dll/cpl/desk/lang/bg-BG.rc

```diff
@@ -115,7 +115,7 @@ BEGIN
     CONTROL "&Използване на следния похват за изглаждане ръбовете на екранните шрифтове", IDC_EFFAPPEARANCE_SMOOTHING,//FIXME: text truncation. I used "&Изгладете ръбовете на екранните шрифтове чрез" for the backport
             "button", BS_AUTOCHECKBOX | WS_TABSTOP, 10, 42, 267, 19
     COMBOBOX IDC_EFFAPPEARANCE_SMOOTHINGTYPE, 20, 62, 95, 19, CBS_DROPDOWNLIST |
-             CBS_HASSTRINGS | CBS_SORT | WS_VSCROLL | WS_TABSTOP
+             CBS_HASSTRINGS | WS_CHILD | WS_VSCROLL | WS_TABSTOP
```

Comment:
The ws_child is also redundant, it's automatically included in principle. Just follow what the other combobox does also.

Reply:
@Hermes your comment is confusing me a bit, because it does lead me into 2 different directions. I do admit that I don't know why the WS:CHILD is there. On one hand you say: "Follow what the other comboboxes do" and this is what I did and why I kept it: Because the other comboboxes do it as well (see line 114 e.g.). Then you say in the same sentence "strip it", that feels like a contradiction to me. If you can explain why it is actually not needed, then yes I would strip it (for those both comboboxes). But I would have to trust you here to not break anything. Do you know by which intention it was put here in the first place? And what part makes it implicitly available? Thank you for your answers in advance. If I do understand both answers, then I will strip it.

Reply:
Ok yes sorry, I didn't check myself what the other comboboxes had here in this file. 
But what's true is that this ws_child flag is completely unnecessary. 
I suppose someone added it originally, because when you look at how the compiled resources look like with a resource editor, you'll see:
`CONTROL ....., WS_CHILD | other styles here`
and so they added it there. 
And you may be able to verify this, by not adding this style here, recompiling the module, and open it in reshacker and see how this looks like.

Reply:
I will do this. If it is still there, then I will strip it. Thank you for your suggestion. I do always strive for short code, also in rc.

Reply:
![Unbenannt](https://github.com/user-attachments/assets/e375ca2a-2dae-4bfe-9507-54a58f521d67)

Reply:
You are correct. I double-checked: When removing the explicit WS_CHILD from the 3 comboboxes that do have it, and then recompiling, it is still implicitly added to the binary. I will remove the explicit WS_CHILD from the following 3 comboboxes therefore: IDC_ADVAPPEARANCE_ELEMENT, IDC_EFFAPPEARANCE_ANIMATIONTYPE, and IDC_EFFAPPEARANCE_SMOOTHINGTYPE for all langs, as you suggested.

Reply:
Applied, and recreated the #7399 . Superseded now.